focuspoints.lrdevplugin/DefaultPointRenderer.lua

```diff
@@ -128,6 +135,10 @@ function DefaultPointRenderer.getAutoFocusPoint(metaData)
     focusPointUsed = ExifUtils.findValue( metaData, DefaultPointRenderer.metaPrimaryAFPoint )
   end
 
+  if "(none)" == focusPointUsed then
```

Comment:
exiftool uses "(none)" so it should be good as long as the tool stays the same.  There are other ways of launching exiftool that will produce no output at all if the tag isn't populated.  There are other issues with Pentax and Canon as they use the "AFPointsInFocus" tag, which isn't in this plugin yet.  A good entropy of test files will be needed to build out the EXIF parsing logic.